tsfm_public/toolkit/get_model.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+import yaml
+
+from tsfm_public.models.tinytimemixer import TinyTimeMixerForPrediction
+
+
+def check_ttm_model_path(model_path):
+    if "ibm-granite/granite-timeseries-ttm-r1" in model_path:
+        return 1
+    elif "ibm-granite/granite-timeseries-ttm-r2" in model_path:
+        return 2
+    elif "ibm/ttm-research-r2" in model_path:
+        return 3
+    else:
+        return 0
+
+
+class GetTTM(TinyTimeMixerForPrediction):
```

Comment:
Do we need to derive a new class from TinyTimeMixerForPrediction -- can we make this a simple utility function?

Reply:
we want to have it as class for better abstraction. But it can be a simple  util class. @ajati - do you see a need to inherit from TinyTimeMixerForPrediction

Reply:
No extra benefit for now. But, later if we need to use the some parent variables (e.g., config vars) it can be done. On the other hand, having a utility function is also good, since we need to define only one function. 

For example, we have `load_datasets` function. Similarly, we can have one unified `load_model` function. Let's think which one will be better.

Reply:
I think I like a `load_model` function better -- it would take `context_length`, `prediction_length`  plus `kwargs` that get passed to the model init. We can later generalize to support other model classes when we have them.